ikasaneip/wiretap/src/main/java/org/ikasan/wiretap/model/WiretapFlowEvent.java

```diff
@@ -77,7 +74,7 @@ public WiretapFlowEvent(final String moduleName, final String flowName, final St
         super(moduleName, flowName, componentName, event, expiry);
         super.eventId = eventId;
         this.relatedEventId = relatedEventId;
-        this.eventTimestamp = eventTimestamp;
+        super.timestamp = eventTimestamp;
```

Comment:
shouldnt it just be this.timestamp ?

Reply:
> shouldnt it just be this.timestamp ?

timestamp is on the super class. Can be either or.